src/main/java/edu/byu/cs/dataAccess/SubmissionDao.java

```diff
@@ -32,12 +32,20 @@ public interface SubmissionDao {
     Collection<Submission> getSubmissionsForUser(String netId);
 
     /**
-     * Gets all submissions for the given phase
+     * Gets all latest submissions
      *
-     * @return all submissions for the given phase
+     * @return all latest submissions
```

Comment:
Note that these API specs do not specify which order the submissions should be returned in. Can we expect them to be in ascending or descending order, or could they come to us in any order? It would seem reasonable to define that behavior as part of this API, and if so, the documentation should reflect the decision.

Reply:
Oh great! I didn't see the client side code that was sorting it in the table. Knowing that it's already taken care of in that context resolves all my concerns. 

Great work delivering really cool and well crafted solutions, Dallin!